package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@lifeomic/jupiterone-aws-integration",
+  "version": "1.0.0",
+  "description": "JupiterOne AWS Integration",
+  "main": "index.js",
+  "scripts": {
+    "prettier": "prettier ./**/*.{js,md,json} *.js --write"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lifeomic/jupiterone-aws-integration.git"
+  },
+  "author": "LifeOmic <development@lifeomic.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/lifeomic/jupiterone-aws-integration/issues"
+  },
+  "lint-staged": {
+    "*.{js,md,json}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "homepage": "https://github.com/lifeomic/jupiterone-aws-integration#readme",
+  "devDependencies": {
+    "husky": "^0.14.3",
```

Comment:
I think you need to add `"precommit": "lint-staged"` in `scripts` to get `husky` working properly right?

Reply:
Fixed thanks.